hersh/hersh.cpp

```diff
@@ -6,23 +6,26 @@
 
 using namespace hersh;
 
-const std::string defaultPrompt = "hersh";
+const std::string defaultPrompt = "hersh$ ";
```

Comment:
Shouldn't the default prompt live in the `reader`?

Reply:
Yes, that's better.